theme/less/gouvfr/navbars.less

```diff
@@ -270,17 +274,16 @@
 
         .input-group-btn  {
             width: none;
-
             .btn {
-                background-color: @bg-light;
-                border-radius: 0px;
-                color: #999;
-                border: 1px solid white;
+              background-color: @bg-light;
```

Comment:
This is not the indentation of the rest of the file. Also modify `.banner` styles.